.github/release-drafter.yml

```diff
@@ -5,26 +5,29 @@ categories:
     label: 'feature'
   - title: 'Enhancements'
     label: 'enhancement'
-  - title: 'Documentation'
-    label: 'documentation'
+  - title: 'Deprecations'
+    label: 'deprecation'
   - title: 'Bug Fixes'
     label: 'bug'
+  - title: 'Documentation'
+    label: 'documentation'
   - title: 'Maintenance'
     label: 'maintenance'
-  - title: 'Deprecations'
-    label: 'deprecation'
```

Comment:
Categories have been reordered following https://github.com/GenericMappingTools/pygmt/issues/885#issue-807847528. Do we want to reorder things in the previous release changelogs (i.e. v0.1.0 to v0.3.0) or just keep them as they were?

Reply:
> Categories have been reordered following [#885 (comment)](https://github.com/GenericMappingTools/pygmt/issues/885#issue-807847528). 

Looks good to me.

> Do we want to reorder things in the previous release changelogs (i.e. v0.1.0 to v0.3.0) or just keep them as they were?

No need to reorder the old entries.